config/webpack/dev.js

```diff
@@ -1,20 +1,22 @@
+var fs = require('fs');
 var path = require('path');
 var webpack = require('webpack');
 var postcssAssets = require('postcss-assets');
 var postcssNext = require('postcss-cssnext');
 var stylelint = require('stylelint');
 var ManifestPlugin = require('webpack-manifest-plugin');
 
+const {
```

Comment:
Seems like it breaks the build https://travis-ci.org/barbar/vortigern/builds/193975281